zerver/openapi/python_examples.py

```diff
@@ -1215,8 +1232,9 @@ def test_queues(client: Client) -> None:
     queue_id = register_queue(client)
     deregister_queue(client, queue_id)
 
-def test_server_organizations(client: Client) -> None:
+def test_server_organizations(client: Client, owner_client: Client) -> None:
 
+    update_realm(client)
```

Comment:
That's because, in the python example, we are only trying to change the `digest_weekday` and `signup_notifications_stream_id` both can be changed by an admin. But in the `curl_example,` it tries to update all the attributes that were defined on `zulip.yaml` for the `/realm:patch` and a couple of them can only be changed by an organization owner.
I let the `owner_client` do the changes on `python_examples` just in case someone wanted a different example, it wouldn't break with `Desdemona` for sure but it can break with `Iago`.